### PR TITLE
pretty docs DAGs

### PR DESCRIPTION
**Pull request by siebrenf**
RNA-seq:
image

ATAC-seq:
image

scATAC-seq:
image

### CODE REVIEW

File: docs/scripts/clean_dags.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
+import re
+import sys
+import networkx as nx
+
+rules_to_hide = [
```
Comment (Maarten-vd-Sande) [inline docs/scripts/clean_dags.py:5]: would it make sense to change this into a rules to keep? :innocent:

**Review by Maarten-vd-Sande — APPROVED** [review]

**Comment by Maarten-vd-Sande** [comment]
Should we mention in the docs that this is a "simplified" graph?

File: docs/scripts/clean_dags.py (hunk 1 above)
Comment (siebrenf) [reply docs/scripts/clean_dags.py:4]: yea it would :cry:

Comment (Maarten-vd-Sande) [reply docs/scripts/clean_dags.py:4]: It's also easier if rules get added later down the line :grimacing:

**Pull request merged**